libs/menu-matriarch/shared/data-access-api/src/lib/internal/calculate-tally-change.spec.ts

```diff
@@ -1,103 +0,0 @@
-import { calculateTallyChange } from './calculate-tally-change';
-
-describe('calculateTallyChange', () => {
-  let tally: Record<string, number>;
-
-  beforeEach(() => {
-    tally = {
-      fruits: 3,
-      vegetables: 1,
-      produce: 0,
-    };
-  });
-
-  it('throws an error on decrement or clear when value does not exist in tally', () => {
-    expect(() =>
-      calculateTallyChange({
-        tally,
-        key: 'fake',
-        change: 'decrement',
-      })
-    ).toThrowError(
-      'Cannot decrement or clear value: key is not present in tally'
-    );
-
-    expect(() =>
-      calculateTallyChange({
-        tally,
-        key: 'fake',
-        change: 'clear',
-      })
-    ).toThrowError(
-      'Cannot decrement or clear value: key is not present in tally'
-    );
-  });
-
-  it('returns 1 on increment when value does not exist in tally', () => {
-    expect(
-      calculateTallyChange({
-        tally,
-        key: 'fake',
-        change: 'increment',
-      })
-    ).toBe(1);
-  });
-
-  it('returns 1 on increment when value is 0', () => {
-    expect(
-      calculateTallyChange({
-        tally,
-        key: 'produce',
-        change: 'increment',
-      })
-    ).toBe(1);
-  });
```

Comment:
This test wasn't carried over to the `Tally` class because it's impossible to generate a tally with a value of 0 (everything in the initial array will have a value of at least one, depending on how many times it's present)